inappmessaging/build.gradle

```diff
@@ -21,15 +21,8 @@ android {
     it.java.srcDirs += "src/$it.name/kotlin"
   }
 
-  compileOptions {
-    // Source code java level(Source code of this project).
-    sourceCompatibility JavaVersion.VERSION_1_8
-    // Target JVM level(JVM on Android devices).
-    targetCompatibility JavaVersion.VERSION_1_8
-  }
-
   kotlinOptions {
-    jvmTarget = JavaVersion.VERSION_1_8.toString()
+    jvmTarget = JavaVersion.VERSION_11.toString()
```

Comment:
nit: Why not use the same format as `analytics` module?
`JavaVersion.toVersion(CONFIG.versions.java).toString()`

Reply:
updated